R/DAISIE_DE_logp0.R

```diff
@@ -0,0 +1,68 @@
+#' @name DAISIE_DE_logp0
+#' @title Log-likelihood of a mainland species that colonizes the island
+#' but leaves no descendants
+#'
+#' @description
+#' Computes the log-likelihood of a colonization event where a mainland species
+#' arrives on the island but does not leave any surviving descendants.
+#'
+#' @inheritParams default_params_doc
+#' @return A single numeric value:
+#' \describe{
+#'   \item{logL0b}{Log-likelihood of the scenario where the lineage colonizes the island
+#'   but does not leave any surviving descendants, computed from the ODE solution.}
+#' }
+#'
+#' @examples
+#' # Example model parameters
+#' pars1 <- c(0.2, 0.1, 0.05, 0.02, 0.03)
+#'
+#' # Compute log-likelihood
+#' log_likelihood <- DAISIE_DE_logp0(island_age = 10,
+#'                                  pars1 = pars1,
+#'                                  methode = "lsodes",
+#'                                  reltolint = 1E-12,
+#'                                  abstolint = 1E-12)
+#' print(log_likelihood)
+#'
+#' @export
```

Comment:
```suggestion
#' @noRd
```
Does this need to be exported? Looks like an internal function to me.